config/environments/production.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.configure do
   # Specify environment specific hostname and protocol
-  config.hostname = ENV["HOSTNAME"]
+  config.hostname = "#{ENV["HOSTNAME"]}/api"
```

Comment:
But right now it's /api, and on staging it runs in production mode (so this file is used), so I think this code is correct but we should add a TODO to change this when we get the new DNS record?